### PR TITLE
*: make QPS more realistic

### PR DESCRIPTION
**Pull request by jackysp**
Signed-off-by: Shuaipeng Yu <jackysp@gmail.com>

### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:
The QPS pannel is exactly CPS. Try to fix it.
企业微信截图_20200602131855

### What is changed and how it works?

Proposal: [xxx](url) 

What's Changed:

How it Works:
企业微信截图_20200602132556

### Related changes

- Need to cherry-pick to the release branch

### Check List 

Tests 

- Manual test (add detailed scripts or steps below)
1. start a cluster
1. run sysbench update non index
1. check out the monitor

### Release note 
- No release note.

### CODE REVIEW

File: executor/adapter.go [hunk 1]
```diff
@@ -217,7 +217,6 @@ func (a *ExecStmt) PointGet(ctx context.Context, is infoschema.InfoSchema) (*rec
 		}
 		a.PsStmt.Executor = newExecutor
 	}
-	stmtNodeCounterSelect.Inc()
```
Comment (cfzjywxk) [inline executor/adapter.go:220]: Seems `StmtCtx.StmtType` should be set in this path if this line is removed

Comment (jackysp) [reply executor/adapter.go:220]: So I add https://github.com/pingcap/tidb/pull/17560/files#diff-928cb614ae092ed4c6f349f4c4c991e4R1412 for all prepared statements.

Comment (cfzjywxk) [reply executor/adapter.go:220]: Maybe it's difficult for users to understand `CPS`, or what a `command` means?

Comment (jackysp) [reply executor/adapter.go:220]: emm... yes, but I'm not sure if we should provide more details in the description of the metric.

Comment (jackysp) [reply executor/adapter.go:220]: Add more descriptions...

Comment (cfzjywxk) [reply executor/adapter.go:220]: Do we need another panel to show the real `QPS` or just use `Statement OPS`

Comment (jackysp) [reply executor/adapter.go:220]: I used to try to add a new panel, but there're too many panels in TiDB dashboard.

**Comment by cfzjywxk** [comment]
LGTM

**Pull request merged**